x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/saml/SamlSpMetadataAction.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.core.security.action.saml;
+
+import org.elasticsearch.action.ActionType;
+
+public class SamlSpMetadataAction extends ActionType<SamlSpMetadataResponse> {
+    public static final String NAME = "cluster:monitor/xpack/security/saml/metadata";
```

Comment:
The only problematic thing with this is that users with `manage_saml` would understandably expect to be able to call the API but now they can't. We should add this action name to the MANAGE_SAML_PATTERN in ClusterPrivilegeResolver

Reply:
Added